src/Kusto/test/utils.ps1

```diff
@@ -0,0 +1,24 @@
+function RandomString([bool]$allChars, [int32]$len) {
+    if ($allChars) {
+        return -join ((33..126) | Get-Random -Count $len | % {[char]$_})
+    } else {
+        return -join ((48..57) + (97..122) | Get-Random -Count $len | % {[char]$_})
+    }
+}
+$env = @{}
+function setupEnv() {
+    # Preload subscriptionId and tenant from context, which will be used in test
+    # as default. You could change them if needed.
```

Comment:
Right now, it looks the test env is set up manually. And we need to put them here, otherwise people will not be able to run live/record test unless they set up the test env manually too.